internal/support/notifications/const.go

```diff
@@ -56,6 +56,15 @@ type ConfigurationStruct struct {
 	DBType                        string
 }
 
+type Monitor struct {
```

Comment:
This type will be used by every service, correct? Put it someplace in the internal package hierarchy where it can be shared.

Also, some name besides Monitor. Maybe something more like "telemetry" (definition: the process of recording and transmitting the readings of an instrument.)